websites/T/Tempest Fansub/presence.ts

```diff
@@ -0,0 +1,124 @@
+const presence = new Presence({
+		clientId: "1047102386478534727",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const { pathname } = window.location,
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/bhEsT7P.png",
+		};
+	let url: string;
+	if (location.origin.startsWith("https://"))
+		url = location.origin.replace("https://", "");
+	else url = location.origin;
+	//Dino
+	if (pathname.endsWith("/dinozor-oyunu/"))
+		presenceData.details = "Dino oynuyor.";
+	//Animeler
+	else if (
+		(url === "tempestfansub.com" &&
+			pathname === "/" &&
+			!url.startsWith("manga.")) ||
+		pathname.endsWith("/anime/")
+	)
+		presenceData.details = "Animelere göz atıyor.";
+	//Bir Anime
+	else if (!location.href.endsWith("/anime/") && pathname.includes("/anime/")) {
+		presenceData.details = "Bir animeye bakıyor.";
+		presenceData.state = document
+			.querySelector(
+				"#content > div > div > article > div:nth-child(2) > div > div.infox > h1.entry-title"
+			)
```

Comment:
Shorten your query selectors where possible. Query selectors do not need to have the entire direct path to the element you are selecting.
```suggestion
			.querySelector(
				"h1.entry-title"
			)
```